ProjectBank/Server.Tests/Controllers/UserControllerTests.cs

```diff
@@ -0,0 +1,54 @@
+namespace Server.Tests;
+
+public class UserControllerTests
+{
+    [Fact]
+    public async Task Post_creates_User()
+    {
+        // Arrange
+        var toCreate = new CreateUserDTO();
+        var created = new UserDTO("1", "Alice");
+
+        var repository = new Mock<IUserRepository>();
+        repository.Setup(m => m.CreateUserAsync(toCreate)).ReturnsAsync(created);
+        var controller = new UserController(repository.Object) { GetObjectId = () => "1" };
+
+        // Act
+        var result = await controller.Post(toCreate) as CreatedAtActionResult;
+
+        // Assert
+        Assert.Equal(created, result?.Value);
+        Assert.Equal("Get", result?.ActionName);
```

Comment:
It is not that good practice to have a "magic string" like we do here. Why do we need to assert the request type?